examples/js/loaders/FBXLoader.js

```diff
@@ -3650,15 +3650,11 @@
 
 		getString: function ( size ) {
 
-			var a = new Uint8Array( this.getUint8Array( size ) );
+			var s = THREE.LoaderUtils.decodeText( this.getUint8Array( size ) );
 
-			// for ( var i = 0; i < size; i ++ ) {
+			this.skip( size );
 
-			// 	a[ i ] = this.getUint8();
-
-			// }
-
-			return THREE.LoaderUtils.decodeText( a );
+			return s;
```

Comment:
Why does this PR include `FBXLoader` change?

Reply:
not sure, apparently there was a change when I forked it. I didn't make any intentional changes to it though.

Reply:
Seems like you've added in this commit (perhaps with misoperation or something) https://github.com/mrdoob/three.js/pull/12932/commits/c7ef06d1b69035958ba334b42b6253e4e4f96e6b

`FBXLoader` will revert to the older version with this change. Can you fix by removing this change from this PR?